cmd/opm/registry/serve.go

```diff
@@ -3,33 +3,46 @@ package registry
 import (
 	"context"
 	"database/sql"
+	"errors"
 	"fmt"
 	"net"
 	"os"
+	"path/filepath"
 	"strconv"
 	"time"
 
+	"github.com/h2non/filetype"
+	"github.com/h2non/filetype/matchers"
 	"github.com/sirupsen/logrus"
 	"github.com/spf13/cobra"
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/reflection"
 
 	"github.com/operator-framework/operator-registry/pkg/api"
 	health "github.com/operator-framework/operator-registry/pkg/api/grpc_health_v1"
+	"github.com/operator-framework/operator-registry/pkg/declcfg"
 	"github.com/operator-framework/operator-registry/pkg/lib/dns"
 	"github.com/operator-framework/operator-registry/pkg/lib/graceful"
 	"github.com/operator-framework/operator-registry/pkg/lib/log"
 	"github.com/operator-framework/operator-registry/pkg/lib/tmp"
+	"github.com/operator-framework/operator-registry/pkg/registry"
 	"github.com/operator-framework/operator-registry/pkg/server"
 	"github.com/operator-framework/operator-registry/pkg/sqlite"
 )
 
+const (
+	databaseFlag = "database"
```

Comment:
Real minor nit: Does this need to be a global variable for the registry package? It looks like most of the existing code attempts to get the value of this flag through `cmd.Flags().Get...`.

Reply:
Yeah I should probably either use hardcoded `"database"` everywhere or change other `cmd.Flags().Get` calls to use a const. Whenever I see a string used this way in more than one place, my default is to change it to a const, to avoid typo mistakes and make use of IDE auto-completion.

As an aside, using `cmd.Flags().Get()` is much more cumbersome than just declaring a flags struct and populating fields in the struct, like this:

```go
type serve struct {
	debug bool
	database string
	port string
	...
}

func (s serve) RunE(cmd *cobra.Command, args []string) error {
	// populate from serveFunc, replace `cmd.Flags().Get("<field>")` with s.<field>
}

func newRegistryServeCmd() *cobra.Command {
	s := serve{}
	rootCmd := &cobra.Command{
		...
		RunE: s.RunE,
	}
	cmd.Flags().BoolVar(&s.debug, "debug", ...)
	cmd.Flags().StringVarP(&s.database, "database", ...)
	cmd.Flags().StringVarP(&s.port, "port", ...)
	...
}
```

Reply:
For now, reverted to hard-coded strings to be consistent with how other flags work.

In a separate PR, I may see how hard it would be to migrate the existing cobra usage over to structs with flag fields to avoid this issue entirely.